docs/workflows/lazy-downloading.rst

```diff
@@ -0,0 +1,60 @@
+Lazy Downloading
+================
+
+Pulp can sync content in a few modes: 'immediate', 'on_demand', and 'streamed'. Each provides a
+different behavior on how and when Pulp acquires content. These are set as the `policy` attribute
+of the `Remote` performing the sync. Policy is an optional parameter and defaults to `immediate`.
+
+immediate
+  When performing the sync, download all Artifacts now. Also download all metadata now to create
+  the content units in Pulp, associated with the RepositoryVersion created by the sync. `immediate`
+  is the default.
+
+on_demand
+  When performing the sync, do not download any Artifacts now. Download all metadata now to create
+  the content units in Pulp, associated with the RepositoryVersion created by the sync. Clients
+  requesting content trigger the downloading of Artifacts, which are saved into Pulp to be served to
+  future clients.
+
+  This mode is ideal for saving disk space because Pulp never downloads and stores packages that
+  clients don't need.
+
+streamed
+  When performing the sync, do not download any Artifacts now. Download all metadata now to create
+  the content units in Pulp, associated with the RepositoryVersion created by the sync. Clients
+  requesting content trigger the downloading of Artifacts, which are *not* saved into Pulp. This
+  content will be re-downloaded with each client request.
+
+  This mode is ideal for content that you especially don't want Pulp to store over time. For
+  instance, syncing from a nightly repo would cause Pulp to store every nightly ever produced which
+  is likely not valuable.
+
+
+Does Plugin X Support Lazy?
+---------------------------
+
+Plugins do have to enable support for Lazy, so check with your plugin docs to be sure. See the
```

Comment:
s/check with your plugin/check that plugin's/
s/see the plugin docs/see the plugin writer docs/ +link

Reply:
sounds good. fixed in next push.